assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -36,30 +36,81 @@ import { __ } from '@wordpress/i18n';
 export const TIPS = {
   addBackgroundMedia: {
     title: __('Add background media', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Double click any image or video to enter edit mode. To scale, use the slider. To change the focal point, drag the image or video.',
+        'web-stories'
+      ),
+    ],
   },
   cropSelectedElements: {
     title: __('Crop selected element', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Double click any image or video element to enter edit mode. Then, use the slider to scale the element or move its focal point by dragging it.',
+        'web-stories'
+      ),
+    ],
   },
   cropElementsWithShapes: {
     title: __('Crop elements using shapes', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Select a shape from the Shape menu to create a frame.',
+        'web-stories'
+      ),
+    ],
   },
   safeZone: {
     title: __('Stay within the safe zone', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Elements work best when you keep them within the safe zone. Outside of the safe zone, elements like links and buttons may get cropped out or not function properly.',
+        'web-stories'
+      ),
+    ],
   },
   previewStory: {
     title: __('Preview your Web Story', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Use Preview Mode to view your Story before publishing.',
+        'web-stories'
+      ),
+      __(
+        'Make sure buttons, links, logos, and text aren’t too close to the edge of your screen and that no Story element overlaps with the navigation bar.',
+        'web-stories'
+      ),
+    ],
   },
   addLinks: {
     title: __('Add links to design elements', 'web-stories'),
-    description: __('', 'web-stories'),
+    description: [
+      __(
+        'Go to the Design tab. Scroll down to Links and enter a URL. To remove a link, select the “X” to the right of URL box.',
```

Comment:
it RTL would this be on the _left_ of the URL box?

Reply:
most likely. Do we have any translation fns where we could specify that it should be left in RTL?

@swissspidy

Reply:
You could do 

```js
const { cdnURL, isRTL } = useConfig();

...

description: isRTL ? __('...') : __('...')`
```

But generally speaking I would highly recommend not using any such directions in texts. For i18n and a11y reasons. "Scroll down" and "to the right of" doesn't say much for screen reader users.

I'd use something like

```
Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" button.
```

Heck, even this isn't great because "X" doesn't say much and the actual button is hopefully announced differently to screen readers.

So maybe something like this:

```
Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" (Clear) button.
```

Could even make the `(Clear)` only visible  to screen readers by using `TranslateWithMarkup`

Reply:
Let's clear this with design, but i think @swissspidy brings up a great point.

Reply:
Design agrees that the above is a good thing to implement, thanks @swissspidy ! 

@littlemilkstudio Let's update to use: Go to the Links section under the Design tab and enter a URL. To remove a link, select the "X" (Clear) button.

where the (Clear) shows up for just screen readers.